Add overrides from resource step (#1000)
kyma-incubator/compass: docs/kyma-environment-broker/03-06-runtime-overrides.md

# Set overrides for Kyma Runtime

You can set overrides to customize your Kyma Runtime. To provision a cluster with custom overrides, add a Secret or a ConfigMap with a specific label. Kyma Environment Broker uses this Secret and/or ConfigMap to prepare a request to the Runtime Provisioner.

> Overrides can be either global or specified for a given component. In the second case, use the ```component: {"COMPONENT_NAME"}``` label to indicate which component it refers to. Create all overrides in the `compass-system` Namespace.

Reviewer: ```suggestion
Overrides can be either global or specified for a given component. In the second case, use the `component: {"COMPONENT_NAME"}` label to indicate the component. Create all overrides in the `compass-system` Namespace.
```

# Set overrides for provisioning runtime

> To provision cluster with custom overrides you can add `Secret` or `ConfigMap` with specific label.

Reviewer: ```suggestion
You can set overrides to customize your Kyma Runtime. To provision a cluster with custom overrides, add a Secret or a ConfigMap with a specific label. Kyma Environment Broker uses this Secret and/or ConfigMap to send a request to the Runtime Provisioner.
```

> Overrides can be global or for concrete components. In the second case, an appropriate label `component` should be used to indicate to which component it refers.

Reviewer: ```suggestion
Overrides can be either global or specified for a given component. In the second case, use the `{COMPONENT_NAME}` label to indicate which component it refers to. Create all overrides in the `compass-system` Namespace.

>**NOTE:** One Secret or ConfigMap can contain either global overrides or overrides for one specific component. If you want to add overrides for more components, you must create an appropriate number of Secrets or ConfigMaps.
```

> All overrides have to be in `compass-system` namespace.

Reviewer: ```suggestion

```

> Examples are given below:

Reviewer: ```suggestion
See the examples:
```

> 1. ConfigMap with global overrides:

Reviewer: ```suggestion
- ConfigMap with global overrides:
```

    ```yaml
    apiVersion: v1
    kind: ConfigMap
    metadata:
      labels:
        provisioning-runtime-override: "true"
      name: global-overrides
      namespace: compass-system
    data:
      global.enableAPIPackages: "true"
    ```  

> 2. Secret with overrides for `core` component:

Reviewer: ```suggestion
- Secret with overrides for the `core` component:
```

    ```yaml
    apiVersion: v1
    kind: Secret
    metadata:
      labels:
        component: "core"
        provisioning-runtime-override: "true"
      name: core-overrides
      namespace: compass-system
    data:

>       database.password: YWRtaW4xMjMK

Reviewer 2: I was hoping for some easter egg 😥

Reviewer: 🥚 🐣 😄
